meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -403,6 +403,36 @@ WKS_FILE:sota:imx8mp-lpddr4-evk-ebbr ?= "efidisk-sota.wks.in"
 # imx8mp comes with an AzureWave CM276MA PCIe Card
 MACHINE_FEATURES:append:imx8mp-lpddr4-evk = " nxp89xx mxm-mwifiex-load"
 
+# iMX8MN
+UBOOT_SIGN_ENABLE:sota:mx8mn-generic-bsp = "1"
+IMX_DEFAULT_BOOTLOADER:mx8mn-generic-bsp ?= "u-boot-fio"
+UBOOT_DTB_LOADADDRESS:mx8mn-generic-bsp = "0x43000000"
+PREFERRED_PROVIDER_virtual/trusted-firmware-a:mx8mn-generic-bsp ?= "imx-atf"
+EXTRA_IMAGEDEPENDS:append:mx8mn-generic-bsp = "virtual/trusted-firmware-a"
+UBOOT_CLASSES:mx8mn-generic-bsp = "uboot-fitimage"
+ATF_BINARY:mx8mn-generic-bsp = "arm-trusted-firmware.bin"
+UBOOT_CONFIG:mx8mn-generic-bsp = ""
+LMP_BOOT_FIRMWARE_FILES:mx8mn-generic-bsp = "imx-boot u-boot.itb"
+IMAGE_BOOT_FILES:mx8mn-generic-bsp = "imx-boot u-boot.itb ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} boot.scr uEnv.txt"
+IMAGE_BOOT_FILES:sota:mx8mn-generic-bsp = "boot.itb"
+WKS_FILE_DEPENDS:append:mx8mn-generic-bsp = " u-boot-default-script"
```

Comment:
this is only a reminder for my future self

depending on which PR is merged first, the other need to include `imx-boot` to `WKS_FILE_DEPENDS` for imx8MN

https://github.com/foundriesio/meta-lmp/pull/779/commits/7f6b03d4b90f1fed2d2df4b887187d789bcb5974

Reply:
Should I go ahead and add that to this PR?

Reply:
yes, please

Reply:
Done